.github/resources/pypiserver/package_upload.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
put this in the github scripts folder: https://github.com/opendatahub-io/data-science-pipelines-operator/tree/main/.github/scripts